charts/github-bot-safe-settings/templates/NOTES.txt

```diff
@@ -0,0 +1,22 @@
+1. Get the application URL by running these commands:
```

Comment:
Please add more docs here, why it is needed, where we should use it and etc. Now it is not clear at all from the first sight

Reply:
I don't think that we need to add here this ingress logics, it confuses

Reply:
resolve cause it is fork